.travis.yml

```diff
@@ -1,11 +1,29 @@
-sudo: false
+dist: trusty
```

Comment:
Any reason we shouldn't use `xenial` or `stretch` here?

Reply:
Travis CI doesn't have them: https://docs.travis-ci.com/user/reference/overview/

Reply:
Wow, ok, more reason to start moving Prometheus over to our new buildkite account. :-)